packages/toolkit/project/gitlab/.gitlab-ci.tmpl

```diff
@@ -0,0 +1,47 @@
+# This file was autogenerated by 10up Toolkit.
+
+image: docker.10up.com/10up-build/deploy-container:latest
+
+include:
+  - project: 10up-build/ci-library
+    ref: trunk
+    file: /main.yml
+
+setup_10up_toolkit:
+  stage: .pre
+  script: |
+    if [ ! -d "toolkit" ] ; then
+      git clone --verbose -b project-command https://github.com/10up/10up-toolkit.git toolkit
+    fi
+    rm -rf 10up-toolkit
+    ln -s toolkit/packages/toolkit/bin/10up-toolkit.js 10up-toolkit
+    cd toolkit
+    npm install
+  artifacts:
+    paths:
+      - toolkit
+      - 10up-toolkit
+
+test_syntax:
+  stage: test
+  script:
+    - php-syntax
+  dependencies: []
+  cache: {}
+
+test_virusscan:
+  stage: test
+  script:
+    - virus-scan
+  dependencies: []
+  cache: {}
+
+build_plugins_and_themes:
+  stage: build
+  script:
+    - ./10up-toolkit project create-payload $CI_COMMIT_REF_NAME
```

Comment:
`npx --yes 10up-toolkit project create-payload $CI_COMMIT_REF_NAME` would make the previous stage to set up toolkit unnecessary as long as node is avaliable on the container.

Reply:
Awesome